modules/apps/segments/segments-web/src/main/java/com/liferay/segments/web/internal/display/context/SegmentsExperienceSelectorDisplayContext.java

```diff
@@ -119,7 +122,14 @@ private SegmentsExperience _fetchSegmentsExperienceFromRequest() {
 		long segmentsExperienceId = ParamUtil.getLong(
 			_httpServletRequest, "segmentsExperienceId", -1);
 
-		if (segmentsExperienceId == -1) {
```

Comment:
Could you add a test for this in SegmentsExperienceSelectorDisplayContextTest.java?, the class is already there, but we need to include a test with SEGMENTS_EXPERIENCE_IDS, this is the one you should add.